src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5768,10 +5799,15 @@ static void addSlur(const Notation& notation, SlurStack& slurs, ChordRest* cr, c
             }
             newSlur->setTick(Fraction::fromTicks(tick));
             newSlur->setStartElement(cr);
+            const QColor color { notation.attribute("color") };
+            if (color.isValid()) {
+                newSlur->setColor(color);
+            }
             const auto pl = notation.attribute("placement");
-            if (pl == "above") {
+            const auto dir = notation.attribute("orientation");
+            if (pl == "above" || dir == "over") {
```

Comment:
I like this way better than my #17842, but it is a different order, here the 1st (placement) wins, in mine the last (orientation) wins, so could you change the order?

Reply:
Or is the difference in slur and tie handling on purpose?

Reply:
In any case, please make it dependant of 
```cpp
 if (configuration()->musicxmlImportLayout()) {
```